automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/Leonardo.scala

```diff
@@ -81,67 +129,182 @@ object Leonardo extends RestClient with LazyLogging {
-                                                                           Option[GcsBucketName],
-                                                                           Option[List[RuntimeError]],
-                                                                           Instant,
-                                                                           Boolean,
-                                                                           Int,
-                                                                           Boolean](
-    "clusterName",
+
+  implicit val clusterDecoder: Decoder[ClusterCopy] =
+    Decoder.forProduct13[ClusterCopy,
+      RuntimeName,
+      GoogleProject,
+      ServiceAccountInfo,
+      RuntimeConfig,
+      ClusterStatus,
+      WorkbenchEmail,
+      LabelMap,
+      Option[GcsBucketName],
+      Option[List[RuntimeError]],
+      Instant,
+      Boolean,
+      Int,
+      Boolean](
+      "clusterName",
+      "googleProject",
+      "serviceAccountInfo",
+      "machineConfig",
+      "status",
+      "creator",
+      "labels",
+      "stagingBucket",
+      "errors",
+      "dateAccessed",
+      "stopAfterCreation",
+      "autopauseThreshold",
+      "patchInProgress"
+    ) { (cn, gp, sa, mc, status, c, l, sb, e, da, sc, at, pip) =>
+      ClusterCopy(cn, gp, sa, mc, status, c, l, sb, e.getOrElse(List.empty), da, sc, at, pip)
+    }
+
+  //removed the Dataproc Instances (don't know what to have for the implicit value)
+  implicit val getRuntimeResponseCopyDecoder: Decoder[GetRuntimeResponseCopy] = Decoder.forProduct15[GetRuntimeResponseCopy,
+      RuntimeName,
+      GoogleProject,
+      WorkbenchEmail,
+      AuditInfo,
+      Option[AsyncRuntimeFields],
+      RuntimeConfig,
+      URL,
+      ClusterStatus,
+      LabelMap,
+      Option[GcsPath],
+      Option[UserScriptPath],
+      Option[UserScriptPath],
+      Option[List[RuntimeError]],
+      Option[UserJupyterExtensionConfig],
+      Int](
+      //TODO Change clusterName to runtimeName in the future. Pending PR
+      "runtimeName",
+      "googleProject",
+      "serviceAccount",
+      "auditInfo",
+      "asyncRuntimeFields",
+      "runtimeConfig",
+      "proxyUrl",
+      "status",
+      "labels",
+      "jupyterExtensionUri",
+      "jupyterUserScriptUri",
+      "jupyterStartUserScriptUri",
+      "errors",
+      "userJupyterExtensionConfig",
+      "autopauseThreshold"
+    ) { (rn, gp, sa, ai, arf, rc, pu, status, l, jeu, jusu, jsusu, e,  ujec, at) =>
+    //Figure this out
+
+      GetRuntimeResponseCopy(rn, gp, sa, ai, arf, rc, pu, status, l, jeu, jusu, jsusu, e.getOrElse(List.empty), ujec, at)
+    }
+
+  implicit val listRuntimeResponseCopyDecoder: Decoder[ListRuntimeResponseCopy] = Decoder.forProduct14(
+
+    "id",
+    "runtimeName",
     "googleProject",
-    "serviceAccountInfo",
-    "machineConfig",
+    "serviceAccount",
+    "asyncRuntimeFields",
```

Comment:
I think this is removed in latest `develop` branch. See https://github.com/DataBiosphere/leonardo/pull/1347/files#diff-9bef6acafa788ef59ee57f5d1618439aL380

Reply:
Removed